main.go

```diff
@@ -14,39 +14,56 @@ import (
 	"github.com/orbs-network/orbs-network-go/config"
 	"github.com/orbs-network/orbs-network-go/instrumentation"
 	"github.com/orbs-network/orbs-network-go/synchronization/supervised"
+	"github.com/orbs-network/scribe/log"
+	"github.com/pkg/errors"
 	"os"
 )
 
 func main() {
-	httpAddress := flag.String("listen", ":8080", "ip address and port for http server")
-	silentLog := flag.Bool("silent", false, "disable output to stdout")
-	pathToLog := flag.String("log", "", "path/to/node.log")
-	version := flag.Bool("version", false, "returns information about version")
+	logger := instrumentation.GetBootstrapCrashLogger()
+	var node *bootstrap.Node
+	func() { // context of bootstrap crash logging
```

Comment:
why do everything in an anonymous func? what's the point?

Reply:
the anonymous function ends when the bootstrap is complete. it is the context of the deferred `recover()` for bootstrap errors